estimagic/optimization/reparametrize.py

```diff
@@ -42,9 +47,9 @@ def reparametrize_to_internal(params, constraints):
             "ignore", message="indexing past lexsort depth may impact performance."
         )
         actually_fixed = internal["lower"] == internal["upper"]
-        internal.loc[actually_fixed, "fixed"] = True
+        internal.loc[actually_fixed, "__fixed__"] = True
 
-    internal = internal.query("~fixed")
+    internal = internal.query("~__fixed__")
 
     assert (internal["value"] >= internal["lower"]).all(), "Invalid lower bound."
```

Comment:
For comparisons, pandas has a very readable api with ``internal["value"].ge(internal["lower"]).all()``. Here are more options: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.gt.html. This would also make it clear, that these are pd.Series instead of Numpy arrays. And if you always use internal.lower, you could save three characters :).

Reply:
Thanks, done.